docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -289,14 +286,251 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Annotation'
-        '400':
-          $ref: '#/components/responses/BadRequest'
-        '404':
-          $ref: '#/components/responses/NotFound'
-        '415':
-          $ref: '#/components/responses/UnsupportedMediaType'
 
   # ---------------------------------------------------------------------------
+  # GET /search - Search annotations
+  # ---------------------------------------------------------------------------
+  /search:
+    get:
+      tags:
+        - annotations
+      summary: Search for annotations
+      parameters:
+        - name: limit
+          in: query
+          description: The maximum number of annotations to return.
+          schema:
+            type: integer
+            minimum: 0
+            default: 20
+            maximum: 200
+        - name: sort
+          in: query
+          schema:
+            description: The field by which annotations should be sorted.
+            type: string
+            enum: [created, updated, group, id, user]
+            default: updated
+        - name: search_after
+          in: query
+          example: 2019-01-03T19:46:09.334539+00:00
+          description: >
+            <p>Define a start point for a subset (page) of annotation search results.</p>
+
+            <p>Working against the sorted, full set of annotation records matching the current
+            search query, the service will examine the values present in the field by which
+            annotations are sorted (i.e. `sort`). The returned subset
```

Comment:
I notice that text in backticks is not rendered as `<code>` blocks in this context.

Reply:
It's looking correctly-rendered to me, unless I misunderstand:

![image](https://user-images.githubusercontent.com/439947/54141305-c6a40100-43fb-11e9-987d-4e09b56b072b.png)